third_party/libunwind/libunwind.BUILD

```diff
@@ -22,18 +22,12 @@ out_files = [
 ]
 
 exports_files([
```

Comment:
Since all the libunwind files are removed, do we still need this `exports_files` and the following `genrule` for `libunwind`?

Reply:
genrule is still needed, but exports_files seems to be unnecessary now.